### PR TITLE
Update readme to include the instruction to use the prebuilt gotpm tool.

### PR DESCRIPTION
**Pull request by qinkunbao**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,20 @@ It contains the following public packages:
 This repository also contains `gotpm`, a command line tool for using the TPM.
 Run `gotpm --help` and `gotpm <command> --help` for more documentation.
 
+### Use prebuilt `gotpm` binary
+
+You can download the binary from a [release](https://github.com/google/go-tpm-tools/releases) directly.
+
+```bash
+curl -L https://github.com/google/go-tpm-tools/releases/download/v0.4.3/go-tpm-tools_Linux_x86_64.tar.gz -o go-tpm-tools.tar.gz
```
Comment (jkl73) [inline README.md:39]: Use placeholder in the url ```https://github.com/google/go-tpm-tools/releases/download/[VERSION]/go-tpm-tools_[arch].tar.gz
`

Comment (KeithMoyer) [reply README.md:39]: Perhaps just point to https://github.com/google/go-tpm-tools/releases ?

**Review by jkl73 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -31,6 +31,18 @@ It contains the following public packages:
...
+You can download the binary from a [release](https://github.com/google/go-tpm-tools/releases) directly.
+
+```bash
+# VERSION: 0.4.4 ARCH: Linux_x86_64
+curl -L https://github.com/google/go-tpm-tools/releases/download/[VERSION]/go-tpm-tools_[ARCH].tar.gz -o go-tpm-tools.tar.gz
+tar xvf go-tpm-tools.tar.gz
+# On COS image, you need to copy the binary into /var/lib/google/
+./gotpm --help
+```
+
```
Comment (kongoshuu) [inline README.md:45]: Shall we also include instructions to load tdx_guest module for Ubuntu images? Something like:
#### TDX
For Ubuntu images to generate TDX quote, the `tdx_guest` module was moved to linux-modules-extra in 1016 and newer kernels. You should install the module, then either manually load it or reboot.
To install the linux-modules-extra package, run:
```bash
sudo apt-get install linux-modules-extra-gcp
```
To manually load the module, run:
```bash
sudo modprobe tdx_guest
```

Comment (jkl73) [reply README.md:46]: added here #426

**Pull request merged**